### ISSUE TITLE
Create models for the Change API

### ISSUE THREAD

**Issue by pkbullock** [pnp/pnpcore#218]
## Category

- [x] Feature request

## Describe the feature

- Add the ability to call the Change APIs
- Unit Tests for calling the APIs
- If there are special requirements or permissions with this model, add to the documentation

## Contributing

We encourage and welcome contributions for the PnP Core SDK, if you are interested, please comment below to engage with the team. If you want to learn about how to get started, please refer to this guide: [https://pnp.github.io/pnpcore/articles/contributor/readme.html](https://pnp.github.io/pnpcore/articles/contributor/readme.html)

The team is constantly improving the guidance on supporting contributors, if you have any feedback or need help please feel free to start a discussion.

**Comment by jansenbe** [comment]
Let's standardize on the Graph delta API's

**Comment by DaleyKD** [comment]
I've never messed with the Change Log stuff in CSOM, but we did do a SPIKE here with the Graph Deltas, and let's just say they didn't give us the warm fuzzies.

First main thing is that there didn't seem to be a way to do a delta on a Site/Web, only on a drive (doc lib).
Second, there's a comment in the docs (can't find it right now) that suggests that a delta token can eventually expire.  I'm not a huge fan of this.

I'm not telling you what to do or even suggest it, just explaining why we decided to continue using the SharePoint CSOM Change Log.
